### PR TITLE
Blacklist IP's table as a parameter and ability to use custom DNS server for the tests.

### PR DESCRIPTION
**Pull request by AdrianMusial**
This PR is intended to provide configurable table of blacklisted IP addresses and ability to use DNS server IP different from 8.8.8.8 for the tests (via DNS_SERVER_IP environment variable).

### CODE REVIEW

File: t/master.t [hunk 1]
```diff
@@ -253,15 +269,17 @@ max_ttl must >= min_ttl (10)
...
+--- main_config
+    env DNS_SERVER_IP=8.8.8.8;
 --- http_config
     lua_shared_dict test_res 1m;
 --- config
     location /t {
         content_by_lua_block {
             local resolver_master = require "resolver.master"
-            local goog_master, err = resolver_master:new("test_res", "google.com", {"8.8.8.8"}, 10, 30, 0)
+            local goog_master, err = resolver_master:new("test_res", "google.com", {"" .. os.getenv("DNS_SERVER_IP")}, 10, 30, 0)
```
Comment (jkeys089) [inline t/master.t]: I'm not totally clear why the string concatenation is needed here?

File: t/master.t [hunk 2]
```diff
@@ -253,15 +269,17 @@ max_ttl must >= min_ttl (10)
...
             ngx.say(err)
-            goog_master, err = resolver_master:new("test_res", "google.com", {"8.8.8.8"}, 10, 30, -1)
+            goog_master, err = resolver_master:new("test_res", "google.com", {"" .. os.getenv("DNS_SERVER_IP")}, 10, 30, -1)
```
Comment (jkeys089) [inline t/master.t]: I'm not totally clear why the string concatenation is needed here?

**Comment by jkeys089** [comment]
@AdrianMusial this looks great! I had just a couple minor comments. Would you also add a default `DNS_SERVER_IP` var in the Makefile?

**Comment by AdrianMusial** [comment]
@jkeys089 Thank you! I have made changes to the Makefile to provide DNS_SERVER_IP variable there. I also removed non intended concatenation which you mentioned in comments above.

**Pull request merged**